.eslintrc

```diff
@@ -0,0 +1,8 @@
+{
+  "parser": "babel-eslint",
+  "extends": "airbnb",
+  "rules": {
+    "comma-dangle": 0,
+    "new-cap": 0
```

Comment:
if you don't like any of the rules that come with airbnb's config you can turn them off like so